Libraries/NativeAnimation/RCTNativeAnimatedModule.m

```diff
@@ -166,9 +173,10 @@ - (dispatch_queue_t)methodQueue
     RCTLogError(@"Not a value node.");
     return;
   }
+
   RCTValueAnimatedNode *valueNode = (RCTValueAnimatedNode *)node;
   valueNode.value = value.floatValue;
-  [valueNode setNeedsUpdate];
+  [_updatedValueNodes addObject:valueNode];
```

Comment:
Yea, not sure about this change either, looks like `_updatedValueNodes` is pretty useless at the moment. We are only calling cleanupAnimationUpdates on it during the animation loop.

I did notice that `setAnimatedNodeValue` was broken anyway and didn't actually update the value if there is no other animation running. Might want to just revert this and fix it separatly.